src/Android/NativeSensorListener.cpp

```diff
@@ -179,6 +179,28 @@ Java_org_xcsoar_NativeSensorListener_onI2CbaroSensor(JNIEnv *env, jobject obj,
                            AtmosphericPressure::Pascal(pressure));
 }
 
+gcc_visibility_default
+JNIEXPORT void JNICALL
+Java_org_xcsoar_NativeSensorListener_onEngineSensors(JNIEnv *env,
+                                                       jobject obj,
+                                                       jint cht_temp,
+                                                       jint egt_temp,
+                                                       jint oat_temp,
+                                                       jint atmospheric_pressure,
+                                                       jint revs_per_sec)
```

Comment:
Bad indent!!

Reply:
It is a multiple of 2. It is the same as for `Java_org_xcsoar_NativeSensorListener_onHeartRateSensor`. What is wrong with it?

Reply:
Aligned with the first parameter.